### PR TITLE
Christina roperto/c01 e04

### PR DESCRIPTION
**Pull request by christina-roperto**
# Resubmitting C01-E04. 

Exercise (C01-E04): 
Student Name: Christina Roperto

Amended the exercise by creating a custom policy with restricted access (least privilege) to only list and read file as suggested.

## Type of change

- [x] Exercise Submission
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/01class/exercises/c01-e04/christina-roperto/c01-e04.txt [hunk 1]
```diff
@@ -1,13 +1,36 @@
...
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": [
+                "s3:GetObject",
+                "s3:ListBucket"
+            ],
+            "Resource": [
+                "arn:aws:s3:::c01-e04",
+                "arn:aws:s3:::c01-e04/README.md"
```
Comment (caiocezart) [inline classes/01class/exercises/c01-e04/christina-roperto/c01-e04.txt]: you could also use /* for allowing access to any files on that path

File: classes/01class/exercises/c01-e04/christina-roperto/c01-e04.txt [hunk 2]
```diff
@@ -1,13 +1,36 @@
...
+            ]
+        }
+    ]
+}
+```
+
 source: 
-https://www.youtube.com/watch?time_continue=13&v=NHAuCWIHevk&feature=emb_logo
+https://aws.amazon.com/blogs/security/use-the-new-visual-editor-to-create-and-modify-your-aws-iam-policies/
```
Comment (caiocezart) [inline classes/01class/exercises/c01-e04/christina-roperto/c01-e04.txt]: do you want to add those links as a reference on our c01-e04 listing? Feel free to open a PR with content suggestion if that helped you to achieve that

File: classes/01class/exercises/c01-e04/christina-roperto/c01-e04.txt (hunk 1 above)
Comment (christina-roperto) [reply classes/01class/exercises/c01-e04/christina-roperto/c01-e04.txt]: That's good to know. I specified only that file as I wanted to make it as limited as possible :)

File: classes/01class/exercises/c01-e04/christina-roperto/c01-e04.txt (hunk 2 above)
Comment (christina-roperto) [reply classes/01class/exercises/c01-e04/christina-roperto/c01-e04.txt]: Done! I added only the second link as the first one is not relevant anymore (granting full access)

**Pull request merged**